### PR TITLE
homepage complete

### PR DESCRIPTION
**Pull request by sameep-s**
Home Page added .

Screenshot (78)

Screenshot (79)

### CODE REVIEW

File: css/index.css [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  width: 100%;
+  flex-wrap: wrap;
+  border: 0.1rem solid var(--dark);
+  overflow-y: scroll;
+  justify-content: space-evenly;
+}
+
+.categories__container .card {
+  width: 25rem;
+  margin: 1rem 1rem 1rem 1rem;
```
Comment (mariyasada) [inline css/index.css:29]: use margin:1rem instead of margin:1rem 1rem 1rem1rem

**Review by mariyasada — APPROVED** [review]
look at the comment

File: css/index.css [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+.main__heading {
+  font-size: 4rem;
+  font-family: var(--font-heading);
+  font-weight: bolder;
+}
+
+.categories__heading {
+  font-size: 2.5rem;
+  font-weight: bolder;
```
Comment (kedark152) [inline css/index.css:15]: try to use variables for font weight, bold bolder etc

File: css/index.css [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+.card-buttons .btn {
+  font-size: 1.8rem;
+  padding: 0.5rem 2rem;
+}
+
+body .footer {
+  height: 25rem;
+  background-color: rgb(19, 18, 18);
+  color: white;
```
Comment (kedark152) [inline css/index.css:40]: use variables for colors

File: css/index.css [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+}
+
+.card-buttons .btn {
+  font-size: 1.8rem;
+  padding: 0.5rem 2rem;
+}
+
+body .footer {
+  height: 25rem;
+  background-color: rgb(19, 18, 18);
```
Comment (kedark152) [inline css/index.css:39]: use variables for bg-color

**Review by kedark152 — APPROVED** [review]
Code well written and UI looks good, just try to use variables as suggested.

File: css/index.css (hunk 2 above)
Comment (sameep-s) [reply css/index.css:15]: sure, i'll add these classes to my utility class.

File: css/index.css (hunk 1 above)
Comment (sameep-s) [reply css/index.css:29]: sure i'll fix that.

**Pull request merged**